src/components/Quotes.js

```diff
@@ -0,0 +1,53 @@
+/*eslint-disable*/
```

Comment:
- [ Required ] You've done well in this project, however, I found this Eslint disable statement :smiley: :smiley: , I think your project would be much better if you could fix the linter errors in this file. I would like to suggest that you delete this line and fix the linter errors using the `npx eslint . --fix` command.